### DISCUSSION TITLE
ccf-validation-tools/Liver/comments/

### DISCUSSION THREAD

**Discussion by giscus** [hubmapconsortium/ccf-validation-tools#286] [General]
# ccf-validation-tools/Liver/comments/

https://hubmapconsortium.github.io/ccf-validation-tools/Liver/comments/

**Comment by emquardokus** [comment]
Liver_v1.2_DRAFT
https://docs.google.com/spreadsheets/d/1U1UFN1m7fkWj-UgYDNTCfGZUv-opzCjEEYypihDjjSQ/edit#gid=1694828397

**Different labels**
FIXED    In row 32, the term CL:0001056 has different name/label in the source ontology. The name/label in the ASCT+B table is human Dendritic cell and the one in the ontology is dendritic cell, human. For reference, the given name/label by SMEs is conventional Dendritic cell. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

FIXED     In row 30, the term CL:0002138 has different name/label in the source ontology. The name/label in the ASCT+B table is lymphatic endothelial cell and the one in the ontology is endothelial cell of lymphatic vessel. For reference, the given name/label by SMEs is hepatic lymphatic endothelial cell. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

FIXED    In row 34, the term CL:0002399 has different name/label in the source ontology. The name/label in the ASCT+B table is CD1c+ myeloid dendritic cell and the one in the ontology is CD1c-positive myeloid dendritic cell. For reference, the given name/label by SMEs is Conventional DC 2. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

FIXED    In row 39, the term CL:0001203 has different name/label in the source ontology. The name/label in the ASCT+B table is CD8 positiver alpha beta memory T cell CD45RO + and the one in the ontology is CD8-positive, alpha-beta memory T cell, CD45RO-positive. For reference, the given name/label by SMEs is CD8+ Liver-Resident Memory T cell. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

FIXED    In row 37, the term CL:0000624 has different name/label in the source ontology. The name/label in the ASCT+B table is CD4 positive alpha beta and the one in the ontology is CD4-positive, alpha-beta T cell. For reference, the given name/label by SMEs is CD4+T cell. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

FIXED    In row 38, the term CL:0000798 has different name/label in the source ontology. The name/label in the ASCT+B table is gamma delta T cell and the one in the ontology is gamma-delta T cell. For reference, the given name/label by SMEs is gamma delta T cell. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

FIXED    In row 24, the term CL:0000526 has different name/label in the source ontology. The name/label in the ASCT+B table is afferent neuron cell and the one in the ontology is afferent neuron. For reference, the given name/label by SMEs is neuron cell. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

**Blank ontology ID**

1) Substitute leukocyte	CL:0000738 for immune cell which is listed as exact synonym 
In row 15, no term id was found for the name/label Immune cell.
2. ADD nerve	UBERON:0001021 In row 24, no term id was found for the name/label nerve.
3. ADD hepatic portal fibroblast	CL:0009100 In row 27, no term id was found for the name/label portal fibroblast.
4. ADD lymphatic vessel	UBERON:0001473 
5. ADD CT/1 CD8-positive, alpha-beta memory T cell CL:0000909 to follow pattern of recently added lung resident similar cell type In row 36, no term id was found for the name/label CD8 TRm (residence memory).
6. ADD CT/1 natural killer cell CL:0000623 In row 41, no term id was found for the name/label Eomes low Liver NK cell.
7. ADD CT/1natural killer cell CL:0000623 In row 42, no term id was found for the name/label TBet low Liver NK cell.

**Blank ontology ID missing parent**

1. ADDED leukocyte	CL:0000738 since "immune cell" is exact synonym in ontology    In row 15, the term Immune cell without ontology ID has no parent that is from the CL ontology.

2. ADDED new CL term hepatic portal fibroblast	CL:0009100   In row 27, the term portal fibroblast without ontology ID has no parent that is from the CL ontology.

2. ADD CT/1 CD8-positive, alpha-beta memory T cell CL:0000909 to follow pattern of recently added lung resident similar cell type     In row 36, the term CD8 TRm (residence memory) without ontology ID has no parent that is from the CL ontology.

 3.   ADD CT/1 CD8-positive, alpha-beta memory T cell CL:0000909 to follow pattern of recently added lung resident similar cell type  In row 41, the term Eomes low Liver NK cell without ontology ID has no parent that is from the CL ontology.

4.  ADD CT/1 CD8-positive, alpha-beta memory T cell CL:0000909 to follow pattern of recently added lung resident similar cell type    In row 42, the term TBet low Liver NK cell without ontology ID has no parent that is from the CL ontology.

**Comment by aleixpuigb** [comment]
New terms in the ontology:
Liver : endothelial cell of central vein of liver (CL:4033061)

**Comment by emquardokus** [comment]
**Liver v1.3: https://docs.google.com/spreadsheets/d/14UaKKGzb8QUUj6LgGlOyHwzz2G3RrLprPPKzm7fLt2E/edit#gid=1694828397**

**Different label**
REMOVED SPACE at the end of label to fix. The term CL:0000909 has a different name/label in the source ontology in the following 1 row 36. The name/label in the ASCT+B table is CD8-positive, alpha-beta memory T cell _ and the one in the ontology is _CD8-positive, alpha-beta memory T cell. For reference, the given name/label by SMEs is CD8-positive, alpha-beta memory T cell . Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.
